lib/tasks/auto_annotate_models.rake

```diff
@@ -5,49 +5,49 @@
 # NOTE: to have a dev-mode tool do its thing in production.
 if Rails.env.development?
 
-  # rubocop:disable BlockLength
-  task :set_annotation_options do
+  # rubocop:disable Metrics/BlockLength
```

Comment:
Disable this cop

Reply:
Sure, disabling Metrics/BlockLength. For this file, it was auto-generated (presumably the people doing rake knows that it breaks rubocop's cops too)